### ISSUE TITLE
[icon-request] icon for symbolic links based on folder attributes not name/extension

### ISSUE THREAD

**Issue by goreish** [vscode-icons/vscode-icons#1739]
- [ x] I'm sure this issue is not a _duplicate_.
- [x ] I'm submitting an icon request:
  - Type: `symlink`
  - Icon Name: `symlink`
  - Sample original Icon: ![symlink](https://goo.gl/images/P4AqdH)
  - Extensions: `
![symlink-directory](https://user-images.githubusercontent.com/22485422/46451134-99b33a00-c749-11e8-961f-7415ccfb56b9.png)
`
  - Filenames: ``
  - Language ids: ``

- Language Extensions:
- More info:
Working with node modules one often symlink other modules in development and it will be helpful to easily see which modules are linked vs installed.

**Comment by jens1o** [comment]
I don't think this is possible, yet. I think you need to propose this functionality at vscode itself.

@vscode-icons/development Is that correct?

**Comment by JimiC** [comment]
Yeah. Not possible atm.

**Comment by goreish** [comment]
ok thanks for clarifying.

**Comment by jens1o** [comment]
@goreish Could you link the vscode issue you created here, so we can track its status?

**Comment by goreish** [comment]
Found there was an issue already for this https://github.com/Microsoft/vscode/issues/11539 
They added a small arrow at the opposite end of a folder icon and when hovering over it it tells you it is a symbolic link. 
It is not very clear but it does the job when it works.  I noticed it is not very reliable because if folders are changed to symbolic links after a project is loaded it is not detected. Need to re-load the project.
